pkgs/tools/graphics/sanjuuni/default.nix

```diff
@@ -36,7 +36,8 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     homepage = "https://github.com/MCJack123/sanjuuni";
-    description = "A command-line tool that converts images and videos into a format that can be displayed in ComputerCraft";
+    description =
+      "A command-line tool that converts images and videos into a format that can be displayed in ComputerCraft";
```

Comment:
```suggestion
    description = "A command-line tool that converts images and videos into a format that can be displayed in ComputerCraft";
```

Reply:
This formatting was added by nixpkgs-fmt; I'm going to leave it.

Reply:
I just tried that myself and nixpkgs-fmt does not do this, only nixmft and maybe possible alejandra have outdated idea of 80 character line lengths.

Reply:
Whoops, turns out I used the wrong formatter. I've reformatted using the _real_ *nixpkgs-fmt*.